tests/expectations/metrics/test_metrics_util.py

```diff
@@ -272,3 +274,149 @@ def test_get_unexpected_indices_for_multiple_pandas_named_indices_named_unexpect
         "Error: The list of domain columns is currently empty. Please check your "
         "configuration."
     )
+
+
+def test_get_dbms_compatible_metric_domain_column_kwargs_column_not_found():
+    test_column_names: list[str] = [
+        "actors",
+        "artists",
+        "athletes",
+        "business_people",
+        "healthcare_workers",
+        "engineers",
+        "lawyers",
+        "musicians",
+        "scientists",
+        "literary_professionals",
+    ]
+
+    with pytest.raises(gx_exceptions.InvalidMetricAccessorDomainKwargsKeyError) as eee:
+        _ = get_dbms_compatible_metric_domain_kwargs(
+            metric_domain_kwargs={"column": "non_existent_column"},
+            batch_columns_list=test_column_names,
+        )
+    assert (
+        str(eee.value)
+        == f'Error: The column "non_existent_column" in BatchData does not exist.'
+    )
+
+
+def test_get_dbms_compatible_metric_domain_column_kwargs(sa):
+    quoted_column_name: sqlalchemy.quoted_name = sqlalchemy.quoted_name(
+        value="travel_agents", quote=True
+    )
+    test_column_names: list[str] = [
+        "ACTORS",
+        "ARTISTS",
+        "ATHLETES",
+        "BUSINESS_PEOPLE",
+        "HEALTHCARE_WORKERS",
+        "ENGINEERS",
+        "LAWYERS",
+        "MUSICIANS",
+        "SCIENTISTS",
+        "LITERARY_PROFESSIONALS",
+        quoted_column_name,
+    ]
+
+    metric_domain_kwargs: dict
+
+    metric_domain_kwargs = get_dbms_compatible_metric_domain_kwargs(
+        metric_domain_kwargs={"column": "ACTORS"},
+        batch_columns_list=test_column_names,
+    )
+    assert metric_domain_kwargs["column"] == "ACTORS"
+
+    metric_domain_kwargs = get_dbms_compatible_metric_domain_kwargs(
+        metric_domain_kwargs={"column": quoted_column_name},
+        batch_columns_list=test_column_names,
+    )
+    assert metric_domain_kwargs["column"] == quoted_column_name
+
+    metric_domain_kwargs = get_dbms_compatible_metric_domain_kwargs(
+        metric_domain_kwargs={"column": "travel_agents"},
+        batch_columns_list=test_column_names,
+    )
+    assert metric_domain_kwargs["column"] == quoted_column_name
+    assert metric_domain_kwargs["column"] != "TRAVEL_AGENTS"
+
+
+def test_get_dbms_compatible_metric_domain_column_pair_kwargs():
+    quoted_column_name: sqlalchemy.quoted_name = sqlalchemy.quoted_name(
+        value="travel_agents", quote=True
+    )
+    test_column_names: list[str] = [
+        "ACTORS",
+        "ARTISTS",
+        "ATHLETES",
+        "BUSINESS_PEOPLE",
+        "HEALTHCARE_WORKERS",
+        "ENGINEERS",
+        "LAWYERS",
+        "MUSICIANS",
+        "SCIENTISTS",
+        "LITERARY_PROFESSIONALS",
+        quoted_column_name,
+    ]
+
+    metric_domain_kwargs: dict
+
+    metric_domain_kwargs = get_dbms_compatible_metric_domain_kwargs(
+        metric_domain_kwargs={"column_A": "ACTORS", "column_B": quoted_column_name},
+        batch_columns_list=test_column_names,
+    )
+    assert (
```

Comment:
@anthonyburdi I think that the `AND` is because they are part of the `MetricDomainTypes.COLUMN_PAIR` domain type.  I see them as key going together -- or am I missing the obvious here?  Thanks.

Reply:
@anthonyburdi The assertions have been split -- thanks!